clang-tools-extra/test/clang-tidy/checkers/readability/else-after-return-if-consteval.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %check_clang_tidy -std=c++20 %s readability-else-after-return %t 
```

Comment:
While `consteval` was added in C++20, `if consteval` is a C++23 feature. Please use `-std=c++23`
https://en.cppreference.com/w/cpp/language/if#Consteval_if